tests/core/data_structures/test_networks.py

```diff
@@ -79,19 +80,21 @@ def crandn(
     return (rng.standard_normal(size) + 1j * rng.standard_normal(size)) / np.sqrt(2)
 
 
-def random_mps(shapes: list[tuple[int, int, int]]) -> MPS:
+def random_mps(shapes: list[tuple[int, int, int]], normalize: bool = True) -> MPS:  # noqa: FBT001, FBT002
```

Comment:
Instead of ignoring the linter, you could also just fix the underlying reason. 😉
Ruff has quite good documentation for its rules.
See https://docs.astral.sh/ruff/rules/boolean-type-hint-positional-argument/#legend in this case.
Just make this a keyword only argument.
```suggestion
def random_mps(shapes: list[tuple[int, int, int]], *, normalize: bool = True) -> MPS:
```

Reply:
Honestly, I just disagree with the reasoning of this being a problem as given in the documentation in this specific instance. Additionally, I hate the solution with the *, as it makes things unnecessarily harder to read. But I also don't care enough to argue about it, if you insist on making this change.

Reply:
I wouldn't really insist on it.
I think it is just much easier to read and grasp a function call like
```
mps = random_mps(shapes, normalize=True)
```
Compared to
```
mps = random_mps(shapes, True)
```

It is not at all clear from the call site what the Boolean value should mean in the second case. So I personally very much agree with the reasoning of this particular rule.
And given how simple the solution is, I'd just go with it.
I totally see that modern IDE support has generally improved the second situation. However, as I frequently look at and review code when I am not in an IDE with all the convenience features, these small details can really make a difference.